lib/puppet/util/windows/service.rb

```diff
@@ -296,7 +296,7 @@ def exists?(service_name)
     # Start a windows service
     #
     # @param [:string] service_name name of the service to start
-    def start(service_name)
+    def start(service_name, timeout)
```

Comment:
Nit: Should `timeout` be an option instead of a parameter? I usually like to think of the `options` hash as specifying optional config, which seems to be what the timeout is within this context.

Reply:
timeout is no longer optional to the service util module. the default timeout comes from the type now, so it didn't make sense to also let the util module have an optional timeout.

Reply:
Shouldn't the util be treated independently of the type though?  In that case, the timeout is optional. I think re-using the constant here should be ok to maintain that independence.

Reply:
That's a good point, I will update